.travis.yml

```diff
@@ -10,11 +9,19 @@ before_install:
     - git clone --quiet --depth 1 https://github.com/minrk/travis-wheels travis-wheels
     - 'if [[ $GROUP != js* ]]; then COVERAGE=""; fi'
 install:
+    - pip install pip --upgrade
     - pip install "setuptools>=18.5"
-    - pip install -f travis-wheels/wheelhouse -e file://$PWD#egg=ipython[test]
+    - pip install  --upgrade --find-links travis-wheels/wheelhouse $EXTRA_PIP_ARGS -e file://$PWD#egg=ipython[test] $EXTRA_DEPS
     - pip install codecov
 script:
     - cd /tmp && iptest --coverage xml && cd -
+
+matrix:
+    include:
+      - python: 3.5
+        env: EXTRA_DEPS='matplotlib pillow pandas numpy' EXTRA_PIP_ARGS='--find-links http://ccdd0ebb5a931e58c7c5-aae005c4999d7244ac63632f8b80e089.r77.cf2.rackcdn.com --trusted-host ccdd0ebb5a931e58c7c5-aae005c4999d7244ac63632f8b80e089.r77.cf2.rackcdn.com'
```

Comment:
I wouldn't bother installing these on Travis until they provide manylinux builds on PyPI, otherwise it's just more stuff for us to maintain.